python/ray/rllib/agents/impala/vtrace_policy_graph.py

```diff
@@ -216,9 +216,8 @@ def __init__(self,
         # Setup the policy
         dist_class, cat_logit_dim = ModelCatalog.get_action_dist(
             action_space, self.config["model"],
-            dist_type=self.config["dist_type"])
-        if self._is_discrete:
-            logit_dim = cat_logit_dim
+            dist_type=self.config['dist_type'])
+        logit_dim = cat_logit_dim or logit_dim
```

Comment:
This should stay behind `if self._is_discrete:`, or to be guarded differently. Point is, we don't want to take logit_dim from multicategorical distribution but from hidden observation shape.